test/functional/master/test_api_version_requests.py

```diff
@@ -11,10 +11,10 @@ def _build_accept_header_with_api_version(self, version: int):
         return {header: value}
 
     @genty_dataset(
-        no_accept_header=(None, 1),
+        no_accept_header=(None, 2),
         v1_accept_header=(1, 1),
         v2_accept_header=(2, 2),
-        invalid_version_accept_header=(999, 1),
+        invalid_version_accept_header=(999, 2),
```

Comment:
How come these are changed from `1` to `2`? If there's a `v1` in the URI (which all these tests have at the moment since it's hard coded into our `UrlBuilder`, this is changed in #381 ), then we default to the first version of the API if no other valid version is specified in the request header. See [the design doc](https://github.com/box/ClusterRunner/issues/375) for reference